bundles/org.openhab.binding.androidtv/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -43,9 +43,15 @@
 				<label>Hostname</label>
 				<description>Hostname or IP address of the device</description>
 			</parameter>
-			<parameter name="port" type="integer">
-				<label>Port</label>
+			<parameter name="googletvPort" type="integer">
```

Comment:
Changing the parameter name is a breaking change. Is this really that important? If so:

1.  a breaking change alert should be added to the distro repo
2.  readme has to be adapted
3. i18n plugin has to be run to generate translation for defautl language

Same for the `googletvPort` parameter.

Reply:
Unfortunately yes this is required and also fixes a bug. If the port is set on a Shield (or eventually a Philips) that is not default it will use it for both protocols this breaking at least one.   I'll update as requested.